src/Core/BoundedContexts/Headquarters/WB.Core.BoundedContexts.Headquarters/Services/Internal/SystemLog.cs

```diff
@@ -149,6 +152,12 @@ public void WorkspaceUserUnAssigned(string userName, ICollection<string> workspa
             this.Append(LogEntryType.WorkspaceUserUnAssigned, userName, string.Join(", ", workspaces));
         }
 
+        public void CriticalityLevelChanged(QuestionnaireIdentity questionnaire, CriticalityLevel? level)
+        {
+            this.Append(LogEntryType.CriticalityLevelChanged, $"(ver. {questionnaire.Version})", 

```

Comment:
Thire is no information about questionnaire, except version.
Please add title or id